feat: Add Remedy inspired themes

GrayJack opened the pull request:
Closes #2 

I loved how the dark variant turned out!

The light one still have some rough edges that I'll probably deal with in the long run, but I need more usage with light themes in general so I can have a better insight on what pieces and bits I want to modify it more.

Here is some preview images:

<img width="3440" alt="Screenshot 2025-02-27 at 19 15 50" src="https://github.com/user-attachments/assets/45eccf1a-ba60-4318-8d81-e4a0c2b527b6" />

<img width="3440" alt="Screenshot 2025-02-27 at 19 16 21" src="https://github.com/user-attachments/assets/0afabb58-df95-4a1d-a9b5-c46a09e50b17" />

GrayJack commented:
Hey, @MobiCode48! 

Since you were the one requesting this theme, can you try this branch?

Here is a vsix if you prefer:

[monokai-grayjack-1.4.0-remedy-rc0.vsix.zip](https://github.com/user-attachments/files/19017580/monokai-grayjack-1.4.0-remedy-rc0.vsix.zip)

The zip sha256 should be: 7bfbfbb5e2faf1cd184f81fc4757c033462222e0de290af1af1406483d1f47dd
And the sha256 of the vsix should be : 19b4a2de8bf7e76969020f65ede9e3557606871a7a5564fc332c0a840e393512

MobiCode48 commented:
I'm gonna try this today but thank you so much man

MobiCode48 commented:
![image](https://github.com/user-attachments/assets/e2cc5819-ab7e-414f-9c97-30396d15b8f5)
Finally ! I couldn't wait any longer, the theme is wonderful man thank you so much

MobiCode48 commented:
![image](https://github.com/user-attachments/assets/5754409a-f141-496f-b8b7-9cdda3b020c8)
I guess if you could just put the attributes member to be the default color instead, it would be nice. Otherwise the theme feels complete.  The default one : 
![image](https://github.com/user-attachments/assets/e5d8c8ef-b143-4957-af6e-61cc72284b5f)

MobiCode48 commented:
![image](https://github.com/user-attachments/assets/6f470ef7-b6e3-439b-bbf4-1f96fb898b6a)
The issue that leaves me unsettled with the default is mainly the syntax highlighting for attributes member.

GrayJack commented:
> I guess if you could just put the attributes member to be the default color instead, it would be nice.

THe issue with the default color is that is the same as variables, all my themes differentiate those two color-wise. But I also think it is too much red, I'll find a good substitute

GrayJack commented:
I did some changes, I changed the field/attribute/property (gosh, each language call the same thing differently) to be a slightly redish brown, I moved to all types to use yellow instead of light red and now the function parameters are light red.

Overall everything became more clear in the editor with those changes 

<img width="443" alt="Screenshot 2025-02-28 at 20 16 59" src="https://github.com/user-attachments/assets/7049799c-f43d-4697-84ec-b409dfd5de79" />

<img width="481" alt="Screenshot 2025-02-28 at 20 18 00" src="https://github.com/user-attachments/assets/97743b8f-a3be-4c37-a3fc-7d55d597daa4" />

Here is a updated vsix:

[monokai-grayjack-1.4.0-remedy-rc1.vsix.zip](https://github.com/user-attachments/files/19034144/monokai-grayjack-1.4.0-remedy-rc1.vsix.zip)

MobiCode48 commented:
Nice ! I'll try it out

MobiCode48 commented:
The theme is finally complete man thank you so much
